### PR TITLE
[APM2-795] missing translation

### PR DESCRIPTION
**Pull request by new4762**
#### 1. Objective

After changing language to Thai, some text is still in English
- installment: bank names, and text in dropdown “Choose number of monthly payments”
- card: all fields

Fixing missing translation and 
add payment method is not currently available messages 
after this PR https://github.com/omise/omise-magento/pull/337

**Related information**:
Related issue(s): https://opn-ooo.atlassian.net/browse/APM2-671

#### 2. Description of change

- add installment missing translation
- add card form missing translation
- add error message `payment method is not currently available` for `mobile banking` , `internet banking` ,  `installment`

**🔧 Environments:**

    Platform version: Magento CE 2.2.3
    Omise plugin version: Omise-Magento 2.26.0
    Omise plugin version: Omise-php v2.13
    PHP version: 7.1

**✏️ Details:**

pull this PR and run this following command to update translation

```
bin/magento setup:upgrade;
bin/magento setup:di:compile;
bin/magento setup:static-content:deploy th_TH -f
bin/magento cache:flush;
```

#### 4. Impact of the change
image

image

image

image

#### 5. Priority of change

Normal

#### 6. Additional Notes

Any further information that you would like to add.

### CODE REVIEW

**Review by kan98 — APPROVED** [review]

**Review by hpariwat — APPROVED** [review]

**Review by tookaa2 — APPROVED** [review]

**Review by aashishgurung — APPROVED** [review]

**Review by FhanOmise — APPROVED** [review]

File: i18n/th_TH.csv [hunk 1]
```diff
@@ -21,3 +26,11 @@
 "Payment rejected by card issuer.", "รายการถูกปฏิเสธโดยธนาคารผู้ออกบัตร"
 "Card stolen or lost.", "บัตรหายหรือถูกขโมย"
 "Payer did not take action before charge expiration.", "ผู้ซื้อไม่ทำรายการภายในเวลาที่กำหนด"
+"Month", "เดือน"
+"Year", "ปี"
+"Remember this card", "จดจำบัตร"
```
Comment (som-m) [inline i18n/th_TH.csv:31]: How about "บันทึกบัตร" ka?

Comment (som-m) [inline i18n/th_TH.csv:31]:
```suggestion
"Remember this card", "บันทึกบัตร"
```

**Comment by som-m** [comment]
Can we also add the translation for True Money Phone Number? Just saw that it is missing too. 🙏🏻
image

**Comment by new4762** [comment]
> Can we also add the translation for True Money Phone Number? Just saw that it is missing too. 🙏🏻 image

Sure krub . 
what message should is be na krub 
is it `หมายเลขโทรศัพท์สำหรับ ทรูมันนี่`` or something else  🙏

**Comment by tanawin-opn** [comment]
This message too krub "ใช้บัตรใหม่"
image

**Comment by new4762** [comment]
@som-m @tanawin-opn  here update for more missing translation krub 
image

**Review by jutinapas — APPROVED** [review]

**Review by som-m — APPROVED** [review]

**Pull request merged**